src/UIAutomation/UiaOperationAbstraction/UiaOperationAbstraction.h

```diff
@@ -1768,7 +1771,16 @@ namespace UiaOperationAbstraction
                 }
             }
             auto localValue = std::get<typename LocalType>(m_member);
-            return V_VT(localValue) == WrapperType::c_comVariantType;
+            bool isType = V_VT(localValue) == WrapperType::c_comVariantType;
+            if(!isType) return false;
+            if constexpr(WrapperType::c_comVariantType == VT_UNKNOWN) {

```

Comment:
I'm really sorry for missing this before, but we should put brackets on the next line for style consistency :(

Reply:
Once that's fixed I'm good with this change!